### PR TITLE
Week six

### PR DESCRIPTION
**Pull request by BijayPuma**

### CODE REVIEW

File: Homework6-start/Homework6.xcodeproj/project.pbxproj
Comment (maniramezan) [inline Homework6-start/Homework6.xcodeproj/project.pbxproj:1]: Looks like you're copying and re-using previous folders, git is considering these as renames, making a separate copy could help with preventing this

File: Homework6-start/Homework6/View/NewTaskButton.swift
Comment (maniramezan) [inline Homework6-start/Homework6/View/NewTaskButton.swift:1]: It's recommended not to include any commented code in any PR, you can just delete it

File: Week5Homework
Comment (maniramezan) [inline Week5Homework:1]: Why does this show up as delete? Is this intentional to remove this folder?

**Review by maniramezan — CHANGES_REQUESTED** [review]
Looks like you have also removed "weekfive" folder in this PR, not sure if that was intentional or not. Also, the example missing the default behavior for user to tap each row and be able to edit is not available in this example. Would you mind updating the code sample to also include the ability to tap at each row and edit?

File: Week5Homework
Comment (BijayPuma) [reply Week5Homework:1]: I created 3 branch for week5, I deleted 2(were not working) of them and kept only last one I created "WeekFive".

File: Homework6-start/Homework6/View/NewTaskButton.swift
Comment (BijayPuma) [reply Homework6-start/Homework6/View/NewTaskButton.swift:1]: added navigationlink on each row. Now user can tap each row and be able to edit.

**Review by maniramezan — APPROVED** [review]
Grade: Pass (Meets expectation)

Nice job on cool animation and creating re-usable views

**Pull request merged**